.travis.yml

```diff
@@ -31,6 +34,9 @@ matrix:
     env: WAGTAIL="wagtail>=1.5,<1.6"
   - python: "3.5"
     env: WAGTAIL="wagtail>=1.4,<1.5"
+  exclude:
```

Comment:
With the current config I don't think this exclude is doing anything, since wagtail 2.0 is only being ran against Python 3.6.

Reply:
OK, will try removing that.

Main thing i seem to be hitting now is some django-2 issues...  I had to guess what was going on with the migrations (the call_command), and now there is a weird error about clone (see travis).